Formula/sshguard.rb

```diff
@@ -16,12 +16,22 @@ class Sshguard < Formula
   depends_on "autoconf" => :build
 
   def install
-    system "./configure", "--disable-debug",
-                          "--disable-dependency-tracking",
+    system "./configure", "--disable-dependency-tracking",
                           "--disable-silent-rules",
                           "--prefix=#{prefix}",
-                          "--with-firewall=#{firewall}"
+                          "--sysconfdir=#{etc}"
     system "make", "install"
+    mkdir "etc"
+    cp "examples/sshguard.conf.sample", "etc/sshguard.conf"
+    inreplace "etc/sshguard.conf" do |s|
```

Comment:
This fails if you already have an etc/sshguard.conf that has been rewritten. That etc/"path" is interesting, looks like it creates a new version named ".default" if there is something already there when used with install, but it doesn't change the reference on read for use with inreplace.